src/SFA.DAS.EmployerIncentives.Web/Services/IDataService.cs

```diff
@@ -5,6 +5,6 @@ namespace SFA.DAS.EmployerIncentives.Web.Services
 {
     public interface IDataService
```

Comment:
Rather than dataService can we have separate business specific services like IEligibleApprenticeships with a Get() method?  Just because it's the same API (at the moment) shouldn't dictate our implementation.